borg/logger.py

```diff
@@ -0,0 +1,88 @@
+"""logging facilities
+
+The way to use this is as follows:
+
+* each module declares its own logger, using:
+
+    from .logger import create_logger
+    logger = create_logger()
+
+* then each module uses logger.info/warning/debug/etc according to the
+  level it believes is appropriate. a first conversion was done, but
+  can be revised later on:
```

Comment:
remove this "a first conversion was done ....."

source code lives, no need to point that out.